src/ItemFieldSet.tsx

```diff
@@ -83,12 +83,17 @@ export class ItemFieldSet extends React.Component<IItemFieldSetProps, IItemField
             Remove Item
           </Button>);
       }
+      let autoFocus = false;
+      const itemsLength = this.state.items.length;
+      if ((itemsLength === 1 && itemIndex === 0) || (itemsLength > 1 && itemIndex === itemsLength - 1)) {
```

Comment:
replace this by:

const autoFocus = (itemsLength === 1 && itemIndex === 0) || (itemsLength > 1 && itemIndex === itemsLength - 1);